<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(0, 0, 0);
}

QCheckBox{
color:rgb(168,168,168);
font-size:12pt;
}

QLabel{
color:rgb(168,168,168)
}

QLineEdit{
border: 1px solid rgb(100,100,100);
background-color:rgb(230,230,230);
}

QTextEdit{
border: 1px solid black;
background-color: black;
color:rgb(168,168,168);
}

QRadioButton{
color:rgb(168,168,168)
}

QLCDNumber{
border: 2px solid rgb(230,230,230);
}

QPushButton{
color: rgb(168,168,168);
background-color:rgb(0,0,0);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}

QComboBox{
background-color:rgb(230,230,230);
}

QAbstractItemView {
background-color:rgb(230,230,230);
}</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>3</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Loop #</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_setpoint">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>77</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Setpoint (K): </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_setpoint">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="push_heater">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>Heater On</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Thermometer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>55</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Heater Range</string>
   </property>
  </widget>
  <widget class="QComboBox" name="thermo_comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="range_combo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>75</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Low</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Medium</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>High</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_heaterMode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#comboBox_potInput{
padding-left: 23px
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Closed Loop (PID)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Open Loop (Manual)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Heater Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>150</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>P:</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>3</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#status_label{
color: rgb(144, 140, 9);
}</string>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_P">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>150</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>I:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_I">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>150</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>D:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_D">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>150</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
